pkg/cmd/cli/cmd/deploy_test.go

```diff
@@ -160,6 +173,11 @@ func TestCmdDeploy_retryOk(t *testing.T) {
 		t.Fatalf("expected updated config")
 	}
 
+	sort.Strings(deletedPods)
+	if !reflect.DeepEqual(deletedPods, []string{"deployerpod", "posthook", "prehook"}) {
```

Comment:
util.StringSet would require mutiple (bi-directional) calls to Difference to achieve this. Using sort and string slices is just simpler in this case.